README.md

```diff
@@ -27,10 +27,13 @@ using a simpler and newer stack. It's not up-to-par in terms of features, but it
 > ⚠ When running sensei inside a Docker container, the `--material` is limited
 > to descendants of the working directory.
 
-> ⚠ If you expect to use this alias within Git Bash for Windows, you might need
-> to set `MSYS_NO_PATHCONV` to `1` to disable path conversion on the volume paths.
-> This avoids the `C:/Program Files/Git/...: no such file or directory` type of 
-> errors. 
+> ⚠ If you expect to use this alias within Git Bash for Windows, prepend the
+> `--volume` and `--workdir` options with an additional slash (ie `--volume
+> /$(pwd):/$(basename $(pwd)) --workdir //$(basename $(pwd)`. See [known issues
```

Comment:
Is it really necessary to have `//` for the `--workdir`? This path is interpreted inside the container.

Reply:
To be logic, it should also be doubled after the `:`, no?

Reply:
yes, unfortunately it is necessary. I met the issue.
If I do not put the extra / for workdir param (see below )

```
ppo_p@LAPTOP-L2OF2KFG MINGW64 /c/dev/znk/git_formation/formation-elasticsearch (master)
$ alias sensei='docker container run  --interactive --tty --rm --volume /$(pwd):/$(basename $(pwd))  --workdir /$(basename $(pwd)) --publish   ${SENSEI_PORT:-8080}:${SENSEI_PORT:-8080} --env SENSEI_PORT --cap-add=SYS_ADMIN zenika/sensei'

ppo_p@LAPTOP-L2OF2KFG MINGW64 /c/dev/znk/git_formation/formation-elasticsearch (master)
$ sensei build
docker: Error response from daemon: the working directory 'C:/Program Files/Git/formation-elasticsearch' is invalid, it needs to be an absolute path.
See 'docker run --help'.
```

Reply:
OK.